packages/adapter-pg/src/pg.ts

```diff
@@ -183,8 +183,12 @@ export class PrismaPgAdapter extends PgQueryable<StdClient> implements SqlDriver
   }
 
   async executeScript(script: string): Promise<void> {
-    // TODO: crude implementation for now, might need to refine it
-    for (const stmt of script.split(';')) {
+    const statements = script
+      .split(/;[\t\r\n ]*/)
+      .map(stmt => stmt.trim())
```

Comment:
Although, again, since we're trimming whitespace separately here, and the regex above doesn't handle whitespace before the first statement and before each semicolon anyway, I don't think there's much value in the regex and it could be replaced with just `split(';')`.